README.md

```diff
@@ -2,7 +2,7 @@
 
 Web server for data visualization for the Society of Black Solar Professionals.
 
-## Getting Started
+## Getting Started - You get started right now
```

Comment:
@JakeCorroon What are you doing this is not your best work!

Reply:
You told me to do this? Stop yelling at me